src/components/notebookRenderStrategy.tsx

```diff
@@ -20,7 +20,7 @@ export class NotebookRenderStrategy implements ExpandableNodeRenderStrategy {
 
 	element(): JSX.Element {
 		return (
-			<div className={this.isSelected() ? 'picker-selectedItem picker-item notebook' : 'picker-item notebook'} title={this.notebook.name} onClick={this.onClick.bind(this)}>
+			<div className={this.isSelected() ? 'picker-selectedItem picker-item notebook' : 'picker-item notebook'} title={this.breadcrumbs() + '/' + this.notebook.name} onClick={this.onClick.bind(this)}>
```

Comment:
I wonder if this will affect narrator?

Reply:
We'll have to talk to Matthew about this one. This is currently how it's being done in the Shared Notebook component, so we'll have to take a look at that as well.